plugins/aws-appmesh/plugin/commands.go

```diff
@@ -208,27 +207,73 @@ func (plugin *Plugin) setupIptablesRules(
 	return nil
 }
 
+// setupIngressRules installs iptable rules to handle ingress traffic.
+func (plugin *Plugin) setupIngressRules(
+	iptable *iptables.IPTables,
+	config *config.NetConfig) error {
+	if config.ProxyIngressPort == "" || len(config.AppPorts) == 0 {
```

Comment:
is **||** needed here? The only condition would happen is **&&**.

Reply:
Admittedly this is a little over-protective, but if either one is not set, there will be an odd error message produced by this function (as @ofiliz noted above). In the event the function is used out-of-band, I felt an OR was better, as we only want to allow the ingress rules to be set up if *both* are set.